src/styles/viewer.styl

```diff
@@ -50,6 +51,7 @@
     position        relative
     display         flex
     flex-grow       1
+    max-height      100%
```

Comment:
No, but it's a fair question. For some reason the `width` is never bigger than the container, but I don't really understand why. I don't mind adding a `max-width` "just in case", but I haven't seen a problem without it.